server/game/actions/PlayUnitAction.ts

```diff
@@ -35,7 +35,7 @@ export class PlayUnitAction extends PlayCardAction {
                 controller: player
             }).generateEvent(context.source, context),
             cardPlayedEvent
-        ]);
+        ], true);
```

Comment:
That is an important point. Where do you think including the parameter should be included? PlayCardAction? All the way up in PlayerOrCardAbility? A property of PlayCardContext or AbilityContext? I'm not sure exactly the boundaries of what all these things are used for so I don't know quite where this would fit.

Reply:
I think PlayCardAction would work since it's only relevant for those classes.